cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java

```diff
@@ -60,7 +62,8 @@
  * If anything fails during this process then the keystore reverts to the last successfully written file.
  * The keystore is flushed to the filesystem after every put and delete.
  */
-class FileSecureStore implements SecureStore, SecureStoreManager {
+@Singleton
```

Comment:
will this not work if it is not a singleton? If it will, its better to specify the singleton in the guice module.